Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -39,6 +41,9 @@
     private final DoubleProperty velocityD;
     private final DoubleProperty velocityF;
 
+    private final PIDManager leftPidManager;
```

Comment:
rename to something like `leftVelocityPidManager` or something along those lines - there will be a bunch of PIDManagers in here